specification/intelligentdataexperiences/resource-manager/Microsoft.AadCustomSecurityAttributesDiagnosticSettings/preview/2023-03-01-preview/intelligentdataexperiences.json

```diff
@@ -0,0 +1,486 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-03-01-preview",
+    "title": "intelligentdataexperiences",
+    "description": "Azure Active Directory Custom Security Logs Client.",
```

Comment:
1. The title, description and RP name all should be same/similar, but are different here.

2. Is your RP name "intelligentdataexperiences" or "AadCustomSecurityAttributesDiagnosticSettings" ?